pkgs/development/python-modules/imageio/default.nix

```diff
@@ -3,23 +3,27 @@
 , fetchPypi
 , pillow
 , psutil
+, imageio-ffmpeg
 , pytest
 , numpy
 , isPy3k
+, ffmpeg
```

Comment:
Are you sure that ffmpeg (and imageio-ffmpeg) should be hard-dependencies here? They are optional dependencies from imageio's point of view, mainly because ffmpeg is quite heavy.

Reply:
@almarklein I see your concern, but declaring it as a function parameter in the place you mention isn't really "depending" on it at runtime. It's just used as part of `checkInputs`, to execute the tests.